### DISCUSSION TITLE
Schedule for point release of 0.15.x or 0.16.0?

### DISCUSSION THREAD

**Discussion by aperezbios** [hathach/tinyusb#2288] [General]
HI there,

It's been 256 days since the release of version 0.15.0, and given that there have been many, many commits since then, I'm curious if there are any plans to spin a release any time soon. @hathach ?

**Comment by hathach** [comment]
I would like to but is currently too busy. Will release when I got the time to write the release note

**Comment by hathach** [comment]
0.16.0 is released
